docs/_sass/minima/_base.scss

```diff
@@ -271,11 +271,25 @@ table {
   }
 }
 
-/**
- * Prevents page break to cut through content when converting to pdf
- */
 @media print {
+  /**
+  * Prevents page break from cutting through content when converting to PDF
```

Comment:
Sorry, I should have noted this earlier. Shall we replace `when converting to PDF` with `when printing` (because the change affects all printing, not just converting to PDF)?

Reply:
I have replaced this comment, and the one on line 282 to `when printing`